CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ target_link_libraries(caffe2_detectron_custom_ops caffe2_library)
 install(TARGETS caffe2_detectron_custom_ops DESTINATION lib)
 
 # Install custom GPU ops lib, if gpu is present.
-if (${CAFFE2_USE_CUDA})
+if (${CAFFE2_USE_CUDA} OR ${CAFFE2_FOUND_CUDA})
```

Comment:
@vkhalidov: should this be `if ("${CAFFE2_USE_CUDA}" OR "${CAFFE2_FOUND_CUDA}")` instead?

Reply:
@ir413 The proper way to employ the [`if` statement](https://cmake.org/cmake/help/v3.0/command/if.html) would be
```
if (CAFFE2_USE_CUDA OR CAFFE2_FOUND_CUDA)
```
Then both expressions are treated as variables and their values are checked. The current form would first substitute the variables' values into the expression and then perform the check, which  leads to the correct result as well, but is not the classical way to employ the operator.

If you use double quotes, you variable values are substituted into strings, which would lead to unexpected behavior.

So, to conclude, I would propose to use the `if` operator form as stated above

Reply:
Thanks for the explanation @vkhalidov.  If I remember correctly, using `${CAFFE2_USE_CUDA} OR ${CAFFE2_FOUND_CUDA}` results in an error. Using `CAFFE2_USE_CUDA OR CAFFE2_FOUND_CUDA` sounds good to me.